rasa/shared/utils/validation.py

```diff
@@ -246,19 +246,20 @@ def validate_training_data_format_version(
         if isinstance(parsed_version, LegacyVersion):
             raise TypeError
 
+        if str(version_value) < LATEST_TRAINING_DATA_FORMAT_VERSION:
```

Comment:
Shouldn't we be using `version.parse` for this check?

Reply:
we can actually use `parsed_version` (which is `version.parse(version_value)`)

Reply:
PR: https://github.com/RasaHQ/rasa/pull/10366